src/CentrifugeRouter.sol

```diff
@@ -169,6 +186,52 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         }
     }
 
+    /// @inheritdoc ICentrifugeRouter
+    function cancelRedeemRequest(address vault, address controller) external payable protected {
+        validateController(vault, controller);
+        IERC7540Vault(vault).cancelRedeemRequest(0, controller);
+    }
+
+    /// @inheritdoc ICentrifugeRouter
+    function claimCancelRedeemRequest(address vault, address receiver, address controller) external payable protected {
+        validateController(vault, controller);
+        IERC7540Vault(vault).claimCancelRedeemRequest(0, receiver, controller);
+    }
+
+    // --- Transfer ---
+    /// @inheritdoc ICentrifugeRouter
+    function transfer(address asset, address recipient, uint128 amount) external protected {
```

Comment:
I think here and also in pool mgr, we should actually rename this to `transferAssets`, to be more explicit

Reply:
https://github.com/centrifuge/liquidity-pools/pull/355 put up a bit of a transfer refactor here